server/src/main/resources/reference.conf

```diff
@@ -16,8 +17,8 @@
   }
   ssl {
     enabled=false
-    keyStorePassword=password
-    keyStorePath="/path/to/your/cert"
+    keyStorePassword=changeit
+    keyStorePath="/etc/ssl/certs/selfsigned.jks"
```

Comment:
@lsulak do you have any suggestion for a good comment here?

Reply:
not sure, but perhaps something like this?

// location of Java Keystore (JKS) file with certificate to be used for HTTPS communication; it can be sufficient to use self-signed certificate on testing or development environment or when application runs in an internal network, otherwise certificates are usually obtained from a trusted CA

Reply:
it's a bit wordy but thanks will use it

Reply:
yeah I know, feel free to change it :D